foundation/foundation-mda/src/test/java/com/boozallen/aiops/mda/metamodel/element/PipelineSteps.java

```diff
@@ -765,6 +797,12 @@ public void the_pipeline_is_created_with_the_specified_persist_save_mode() {
         assertEquals("Unexpected persist mode found", "test-mode", persist.getMode());
     }
 
+    @Then("the pipeline is created with the specified persist type {string}")
+    public void the_pipeline_is_created_with_the_specified_persist_save_type(String expectedType) {
+        String realType = pipeline.getSteps().get(0).getPersist().getType();
```

Comment:
q: any worries of a null pointer getting the persist type?

Reply:
That is a risk should a future change in the tests cause the pipeline or its steps to get generated improperly. I can add a not-null assertion with an error message that provides the context for the failure should it happen.